charts/stablesats/templates/stablesats-price-deployment.yaml

```diff
@@ -66,6 +66,7 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: spec.nodeName
+        resources: {}
```

Comment:
missing indentation and stablesats/values.yml

Reply:
fixed